lib/cell.rb

```diff
@@ -0,0 +1,63 @@
+class Cell
+
+  attr_reader :mine, :cleared, :flagged, :adjacent_mines
+
+  def initialize
+    @mine = false
+    @cleared = false
+    @flagged = false
+    @adjacent_mines = 0
+  end
+
+  # This ends the game!
+  def exploded?
+    mine && cleared
+  end
+
+  def place_mine
+    @mine = true
```

Comment:
Interesting use of a method here. Probably makes sense. We used an attr_accessor .... and would dew cell.mine = true etc to adjust it.